readme.txt

```diff
@@ -225,6 +225,7 @@ Remember to always make a backup of your database and files before updating!
 = [5.3.3] TBD =
 
 * Fix - Navigation for the Views will no longer use current browser URL as previous url, preventing problems on shortcodes.
+* Tweak - Include a filter `tribe_events_views_v2_view_page_reset_ignored_params` for prevent certain params to interfere with pagination resetting. [FBAR-222]
```

Comment:
```suggestion
* Tweak - Include the `tribe_events_views_v2_view_page_reset_ignored_params` filter to prevent certain params from interfering with pagination resetting. [FBAR-222]
```